jlrs/src/value.rs

```diff
@@ -84,6 +84,42 @@ thread_local! {
 /// and contains said exception.
 pub type CallResult<'frame, 'data> = Result<Value<'frame, 'data>, Value<'frame, 'data>>;
 
+/// Executes the provided string as Julia code in the context of the Main module. For example:
+/// ```rust
+/// # jlrs::util::JULIA.with(|j| {
+/// # let mut julia = j.borrow_mut();
+/// use jlrs::prelude::*;
+/// 
+/// julia.dynamic_frame(|global, frame| {
+///     let result = jlrs::eval_string(global, frame, "2 + 3")?.unwrap();
+///     assert_eq!(result.cast::<isize>().unwrap(), 5);
+/// 
+///     jlrs::eval_string(global, frame, "increase(x) = x + 1")?.unwrap();
+///     let func = Module::main(global).function("increase")?;
+///     let twelve = Value::new(frame, 12isize).unwrap();
+///     let result = func.call1(frame, twelve)?;
+///     assert_eq!(result.unwrap().cast::<isize>().unwrap(), 13isize);
+/// 
+///     Ok(())
+/// })
+/// .unwrap();
+/// # });
+/// ```
+pub fn eval_string<'base, 'frame, F>(
+    _global: Global<'base>,
```

Comment:
`Global` is intended to be used when the result is a value that doesn't have to be protected from garbage collection, while `Frame` is used when this protection is required. This function doesn't need `Global`.